OracleDatabase/SingleInstance/dockerfiles/19.3.0/setPassword.sh

```diff
@@ -12,7 +12,7 @@
 
 ORACLE_PWD=$1
 ORACLE_SID="`grep $ORACLE_HOME /etc/oratab | cut -d: -f1`"
-ORACLE_PDB="`ls -dl $ORACLE_BASE/oradata/$ORACLE_SID/*/ | grep -v pdbseed | awk '{print $9}' | cut -d/ -f6`"
+ORACLE_PDB="`ls -dl $ORACLE_BASE/oradata/$ORACLE_SID/*/ | grep -v -e pdbseed -e archive_logs | awk '{print $9}' | cut -d/ -f6`"
```

Comment:
Can we use a variable to store the directory name for archive_logs and use that wherever archive_logs is referenced. Just in case we decide to rename this folder.

Reply:
Addressed in the next commit.